evaluation_script.py

```diff
@@ -347,4 +347,4 @@ def main():
     # run_experiment(args.task_path, args.task_id, args.num_exp)
 
 if __name__ == "__main__":
-    main()
\ No newline at end of file
+    main()
```

Comment:
I still see like 5 random newline addition/removal. again, please clean it up. look at the diff.